Games/DO TA/import random.py

```diff
@@ -0,0 +1,56 @@
+import random
+
+random_num=[1, 2, 3, 4, 5, 6 , 7 , 8, 9, 10, 11 ,12 , 13, 14, 15, -1, -2, -3, -4, -5, -6 , -7 , -8, -9, -10]
+
+
+random_exp=[1, 2, 3]
+
+
+print("This is VaIoRant")
+print("Out of 10, how many can you get right?")
+
+def mathfun():
+
+  score1 = 0
+  finished0 = 0
+  
+  while finished0 < 10: 
```

Comment:
Is there a way for the player to quit the game gracefully without terminating the program (via CTRL-C)?